src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md

```diff
@@ -227,19 +224,23 @@ fluid.defaults("fluid.prefs.enactors.tableOfContents", {
         afterTocRender: null,
         onLateRefreshRelay: null
     },
-    listeners: {
-        onCreate: {
+    modelListeners: {
+        toc: {
             listener: "{that}.applyToc",
-            args: "{that}.model.value"
+            args: ["{change}.value"]
```

Comment:
Should be toc. At the line 187, the prefs map links the default primary schema value with "model.toc".